app/src/containers/ConnectedSidebar/tests/index.test.tsx

```diff
@@ -94,13 +115,10 @@ describe('components/ConnectedSidebar', () => {
     wrapper.unmount();
   });
 
-  it('correctly expand location menu', () => {
-    const envModule = require('../../../configs/env');
-    envModule.ENABLE_LOCATIONS = 'true';
-
+  it('correctly expand users menu', () => {
```

Comment:
@ashfahan I thought we were to remove this test cases since its already catered for in  `Test order of menu'` test case?

Reply:
these expansions tests were originally written by @HaseemUlHaq @haseemVD i just replicate them over other sidebar to like fill the gap

@HaseemUlHaq can you guide us better on this ?